backend/es/es_index.py

```diff
@@ -93,6 +95,15 @@ def populate(client, corpus_name, corpus_definition, start=None, end=None):
 
     corpus_server = settings.SERVERS[
         settings.CORPUS_SERVER_NAMES.get(corpus_name, 'default')]
+
+    # set corpus-specific locale
+    try:
+        corpus_locale = settings.CORPORA_LOCALES[corpus_name]
+        if corpus_locale:
+            locale.setlocale(locale.LC_ALL, corpus_locale)
+    except:
+        pass
```

Comment:
This is a catch-all `except` (i.e. it's not checking for specific error types); using `pass` as the body means that the script will not only move past the error but it will be completely undetectable. This is a big no-no, as it can create hidden bugs. In this case, if an unexpected error occurs during the indexing setup, you want to know what happened (and you _want_ the script to break so you can fix it first).

In most cases, a try/except block should look for *specific* errors that are expected under the circumstances. If it's a handler for any kind of error (usually because you're evaluating code that is not under the control of the current script, e.g. imported code, user code, etc.), the program should communicate what happened; usually by writing to the log or stderr.

In this case, it looks the block is meant to catch a `AttributeError` or `KeyError`, so a more specific `except` could be used. Both of those errors could also be avoided by providing appropriate fallbacks with `getattr` / `dict.get` during the lookup on line 101;  so you would not need an except clause at all.